lib/resque/scheduler.rb

```diff
@@ -92,7 +92,7 @@ def load_schedule!
         Resque.schedule.each do |name, config|
           load_schedule_job(name, config)
         end
-        Resque.redis.del(:schedules_changed)
+        Resque.redis.del(Resque.keys[:schedules_changed])
```

Comment:
I was referring to the [proxy method i defined](https://github.com/Shopify/resque-scheduler/pull/8/files#diff-74cb42b5aa9ca904352848778d8e0ce4R112). Did you mean `Resque.redis.keys`?